src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write code here
+const salaryList = [...document.querySelectorAll('li[data-salary]')];
+
+salaryList.sort((a, b) => {
+  const item1 = parseInt(a.getAttribute('data-salary').replace('$', ''));
+  const item2 = parseInt(b.getAttribute('data-salary').replace('$', ''));
```

Comment:
You might  add error handling in case the salary attribute contains invalid values.
```suggestion
  const item1 = parseInt(a.getAttribute('data-salary').replace('$', '')) || 0;
  const item2 = parseInt(b.getAttribute('data-salary').replace('$', '')) || 0;
```